public/walkthroughs/1/walkthrough.adoc

```diff
@@ -63,7 +63,7 @@ This walkthrough describes how  to create such an integration, using OpenShift,
 +
 [subs="attributes+"]
 ---- 
-http://{spring-boot-url}/v2/api-docs
+{route-crud-host}/v2/api-docs
```

Comment:
renders as `http://crud-admin-example-com-1.apps.waterford-9226.openshiftworkshop.com/v2/api-docs`,

However, it sometimes takes a second before it shows that, with `{route-crud-host}/v2/api-docs` visible temporarily